### PR TITLE
[MIG] bibo: Submodule bibo migrated to v15.0

### PR DESCRIPTION
**Pull request by agarzaarvizu**
- Migration of the submodule bibo to v15.0
- credit-management v15.0 added as submodule

### CODE REVIEW

**Comment by agarzaarvizu** [comment]
@desdelinux Done

**Comment by desdelinux** [comment]
LGTM @luisg123v your review, please.

File: .gitmodules [hunk 1]
```diff
@@ -1,4 +1,8 @@
 [submodule "bibo"]
 	path = bibo
 	url = git@git.vauxoo.com:vauxoo/bibo.git
-	branch = 12.0
+	branch = 15.0
+[submodule "credit-management"]
+	path = credit-management
```
Comment (luisg123v) [inline .gitmodules:6]: Youre behind the 15.0 branch by 3 commits, has should be 5abd61e

**Review by luisg123v — CHANGES_REQUESTED** [review]

**Comment by agarzaarvizu** [comment]
@luisg123v Done

**Comment by luisg123v** [comment]
I just noticed sh-tools is missing

**Comment by agarzaarvizu** [comment]
@luisg123v Done

**Comment by luisg123v** [comment]
@agarzaarvizu sh-tools is outdated

**Comment by agarzaarvizu** [comment]
@luisg123v Done

**Review by luisg123v — APPROVED** [review]

**Pull request merged**